ios/PresentationLayer/UIToolkit/Sources/UIToolkit/BaseViews/SwiftUI/Snackbar/InfoErrorSnackHost.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  InfoErrorSnackHost.swift
+//
+//
+//  Created by Filip Wiesner on 15.01.2023.
+//
+
+import Foundation
+import SwiftUI
+
+public enum InfoErrorSnackVisuals: SnackVisuals {
+    case info(message: String, duration: Int = defaultDuration)
+    case error(message: String, actionLabel: String?, duration: Int = defaultDuration)
+    
+    public var message: String {
+        switch self {
+        case .info(message: let message, duration: _): return message
```

Comment:
You can omit the parameter labels: `case .info(let message, _): return message`, or if you wanted more parameters, you could use `case let .info(message, duration): ...`.

Reply:
Thanks, changed